includes/internal/emails/class-email-settings-tab.php

```diff
@@ -171,6 +188,47 @@ private function render_teacher_subtab(): void {
 	 * Render the settings subtab.
 	 */
 	private function render_settings_subtab(): void {
-		echo 'TODO';
+		global $wp_settings_fields;
+
+		$fields_to_display = array_filter(
+			$wp_settings_fields['sensei-settings']['email-notification-settings'] ?? [],
+			function( $field_key ) {
+				return in_array( $field_key, [ 'email_from_name', 'email_from_address' ], true );
+			},
+			ARRAY_FILTER_USE_KEY
+		);
+		$fields_to_display = array_map(
+			function( $field ) {
+				unset( $field['args']['data']['description'] );
+				$class = $field['args']['class'] ?? '';
```

Comment:
Not sure about this, but should this be `$field['args']['data]['class']`?

Reply:
[Nope](https://developer.wordpress.org/reference/functions/do_settings_fields/). Here is the original code for that:
```
$class = '';
if ( ! empty( $field['args']['class'] ) ) {
	$class = ' class="' . esc_attr( $field['args']['class'] ) . '"';
}
```